README.md

```diff
@@ -18,13 +18,14 @@ MyDiary is an online journal where users can pen down their thoughts and feeling
 
 - Users can set and get daily notifications that prompt them to add an entry to their diary
 
-## Technologies
-
-- Nodejs
-- Express
-- Mocha (with Chai as assertion library)
-- Babel (ES6+ to ES5 transpilation)
-- ESLint (with Airbnb style guide)
+## Tech Stack
+
+- [Node.js](https://nodejs.org/)
+- [Express.js](https://expressjs.com/)
+- [PostgreSQL](https://www.postgresql.org/)
```

Comment:
Have you used PostgreSQL in this project, if no; remove it and add it when you have used it. 
The Readme file should be updated alongside the project.